kernel/virtio/virtio_net.c

```diff
@@ -71,8 +71,6 @@ int handle_virtio_net_interrupt(void *arg __attribute__((unused)))
     if (net_configured) {
         isr_status = inb(virtio_net_pci_base + VIRTIO_PCI_ISR);
         if (isr_status & VIRTIO_PCI_ISR_HAS_INTR) {
-            virtq_handle_interrupt(&xmitq);
```

Comment:
Add a comment here to make it clear that this interrupt is just to kick us out of any `solo5_poll()` that may be running?